model/storage/src/main/java/org/keycloak/storage/UserStorageProviderSpi.java

```diff
@@ -81,6 +81,8 @@ public Class<? extends ProviderFactory> getProviderFactoryClass() {
                 .name("evictionDay").type(ProviderConfigProperty.STRING_TYPE).add()
                 .property()
                 .name("cacheInvalidBefore").type(ProviderConfigProperty.STRING_TYPE).add()
+                .property()
+                .name("removeInvalidUsersEnabled").type(ProviderConfigProperty.BOOLEAN_TYPE).add()
```

Comment:
There is method `defaultValue(true)` . Is it possible to use it as well?

Reply:
Thx for the feedback. I had to look into this one, bc I more or less did a copy&pasta from the existing `importEnabled` without understanding the details 100%.

It looks like it would not have any effect if we set the `defaultValue` here, bc the endpoints for the `ComponentResource` are not using it:

* When we create a new LDAP provider, the `List<ProviderConfigProperty> config` from the backend is not used to show the initial form.
There the default values are hardcoded in the React code. 

* When we update an existing LDAP provider, the `List<ProviderConfigProperty> config` is only used to match the keys. ([RepresentationToModel#updateComponent](https://github.com/keycloak/keycloak/blob/faf36dd2f061fe0c0392b5b731f3fdaf7c33c34b/server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java#L1068))

* When we read an existing LDAP provider, the `List<ProviderConfigProperty> config` is not used, only the `MultivaluedHashMap<String, String> config` from the `ComponentModel` itself. ([ModelToRepresentation#toRepresentationWithoutConfig](https://github.com/keycloak/keycloak/blob/faf36dd2f061fe0c0392b5b731f3fdaf7c33c34b/server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java#L1009))